src/MessageBox.Avalonia/ViewModels/Base/AbstractMsBoxViewModel.cs

```diff
@@ -36,13 +33,12 @@ public AbstractMsBoxViewModel(AbstractMessageBoxParams @params)
             CanResize = @params.CanResize;
             ContentTitle = @params.ContentTitle;
             ContentHeader = @params.ContentHeader;
-            ContentMessage = @params.ContentMessage;
             LocationOfMyWindow = @params.WindowStartupLocation;
         }
 
-        public async Task Copy()
-        {
-            await AvaloniaLocator.Current.GetService<IClipboard>().SetTextAsync(ContentMessage);
-        }
+        //public async Task Copy()
```

Comment:
Why is this commented out?

Reply:
@rstm-sf Because there are no references pointing to this method so I decided to comment this out, but not to delete because the developer may need this

Reply:
This is a public method. It is not necessary that it is used in this solution